meta-multimedia/recipes-multimedia/libcamera/libcamera.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Linux libcamera framework"
+SECTION = "libs"
+
+LICENSE = "GPL-2.0 & LGPL-2.1"
+LIC_FILES_CHKSUM = "\
+    file://licenses/gnu-gpl-2.0.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263 \
+    file://licenses/gnu-lgpl-2.1.txt;md5=4b54a1fd55a448865a0b32d41598759d \
+"
+
+SRC_URI = "git://linuxtv.org/libcamera.git;protocol=git"
+
+S = "${WORKDIR}/git"
+SRCREV = "${AUTOREV}"
```

Comment:
Please pin to a known SHA

Reply:
Also set PV to something reasonable (not default 1.0).

Reply:
Added the PV for libcamera pointing to latest commit of Oct2019, which is commited on 27Oct19